src/main/java/fi/aalto/cs/apluscourses/intellij/model/SettingsImporter.java

```diff
@@ -42,9 +43,22 @@
    * @throws IOException If an IO error occurs (e.g. network issues).
    */
   public void importIdeSettings(@NotNull Course course) throws IOException {
-    URL ideSettingsUrl = course.getResourceUrls().get("ideSettings");
+    URL ideSettingsUrl = null;
+
+    if (SystemInfoRt.isWindows) {
+      ideSettingsUrl = course.getResourceUrls().get("ideSettingsWindows");
+    } else if (SystemInfoRt.isLinux) {
+      ideSettingsUrl = course.getResourceUrls().get("ideSettingsLinux");
+    } else if (SystemInfoRt.isMac) {
+      ideSettingsUrl = course.getResourceUrls().get("ideSettingsMac");
+    }
+
     if (ideSettingsUrl == null) {
-      return;
+      ideSettingsUrl = course.getResourceUrls().get("ideSettings");
+
+      if (ideSettingsUrl == null) {
+        return;
+      }
```

Comment:
Nit: this can be rewritten more elegantly as
```
URL ideSettingsUrl = course.getResourceUrls().get("ideSettings");
<chain of if-elses>
if (ideSettingsUrl) {
  return;
}
```